### ISSUE TITLE
[executeur_taches][navigation_manager][path_tracker] Bug possible

### ISSUE THREAD

**Issue by ValentinVERGEZ** [PyroTeam/robocup-pkg#32]
Sur un ordre d'exploration à l’exécuteur de tâches, l’exécuteur a semble-t-il demandé plusieurs déplacements (au moins deux). Il a demandé le deuxième avant que le premier finisse et le robot a d'abord terminé son premier déplacement avant d'attaquer le deuxième.

Problème : le second déplacement ayant été demandé quasiment en même temps que le premier, il avait le même point de départ. Résultat, quand le robot a attaqué son second déplacement, il était très loin du chemin à suivre.

Soit le robot aurait du interrompre tout de suite son premier déplacement pour commencer le second, soit le second aurait dû être demandé après la fin du premier (pour que le chemin généré parte bien de la position du robot).

Bref, à surveiller.

**Comment by tdanel** [comment]
Pas vu depuis

**Comment by ValentinVERGEZ** [comment]
Descend le timeout de déplacement à 5secs, c'est plus probable que tu le vois ainsi (si toujours présent)

**Comment by tdanel** [comment]
Sauf que les chemins ne sont plus bufferisés là donc yaura plus le pb, nan ?

**Comment by ValentinVERGEZ** [comment]
Dans le pathtracker tu veux dire ?
Oui, si c'était le problème et qu'on ne passe pas à côté d'autre chose.

Je veux bien fermer l'issue, au regard du comportement du robot sur des
ordres au clic, il repart bien de sa position.

Ceci dit, est-ce que c'est normal que navigation-manager empile les ordres
plutôt que de les refuser ? A moins que ce soit le noeud de "clic clic" qui
les empile...

Le mar. 14 juin 2016 08:35, Thomas DANEL notifications@github.com a
écrit :

**Comment by vcoelen** [comment]
Pourquoi fermer? Est-ce qu'on est sûr que le problème est réglé. Notamment, j'ai toujours un empilement des ordres au clic. (Éventuellement, on peut ouvrir une issue pour ça, puisque ça n'est pas tout à fait le même problème que l'issue initiale)

**Comment by tdanel** [comment]
Faux problème. On ne va pas cliquer pendant un match ni en simulation.

**Comment by vcoelen** [comment]
Ben ça dépend, c'est juste une question, mais si ça arrive au clic, est-ce qu'on a vraiment aucun risque que ça arrive dans une autre situation?

**Comment by ValentinVERGEZ** [comment]
Oui, c'est pas exclu que ce soit le nœud python (moveRequest_node) qui accumule les demandes, mais si ce n'est pas le cas alors c'est plus grave.

C'est assez facile à déterminer, mais il faut le confirmer avant de fermer cette issue svp.
Je m'ajoute aux assignee pour ce qui est de déterminer si c'est un vrai problème ou un faux.
